src/components/UI/NavMobile.tsx

```diff
@@ -0,0 +1,32 @@
+import { createIcon } from '@chakra-ui/icons';
+import { Box, Flex, IconButton, Menu, MenuButton } from '@chakra-ui/react';
+import Image from 'next/image';
+import { FC } from 'react';
+import logoSrc from '../../public/esp-logo.svg';
+
+const HamburgerIcon = createIcon({
+  displayName: 'HamburgerIcon',
+  viewBox: '0 0 32 25',
+  path: <path fill='#F85858' d='M0 0h32v3H0zM0 11h32v3H0zM0 22h32v3H0z' />
+});
```

Comment:
Probably you have already thought about this but would be good to have all the icons grouped in their own folder. A refactor we can implement in next PRs when you have more icons.

Reply:
Agree, I've moved `<HamburguerIcon />` to `/components/UI/icons`